### ISSUE TITLE
OnPointerEnter fires twice on Android

### ISSUE THREAD

**Issue by ersankavafoglu** [Unity-Technologies/InputSystem#876]
I was testing my class which implements IPointerEnterHandler. In editor OnPointerEnter is fired only once as expected but when I build and try on an actual Android device I noticed that OnPointerEnter gets called twice in the same frame (checked with Time.frameCount).

**Comment by ersankavafoglu** [comment]
EnhancedTouchSupport is enabled btw

**Comment by Rene-Damm** [comment]
Thank you for reporting. We'll take a look.

We're in the process of migrating all our bug tracking to the official Unity bug tracking system (which gives us the benefit of properly integrating ourselves with Unity QA procedures) and turn off GitHub issue tracking once done. I've migrated the case over to FogBugz.

FB Ticket: 1189904
Unity Issue Tracker: https://issuetracker.unity3d.com/product/unity/issues/guid/1189904/

Closing the issue here. No worries, doesn't mean it's forgotten :)
